DigitalLearningSolutions.Web.AutomatedUiTests/AccessibilityTests/BasicAuthenticatedAccessibilityTests.cs

```diff
@@ -13,6 +13,7 @@ public class BasicAuthenticatedAccessibilityTests : AccessibilityTestsBase,
         [Theory]
         [InlineData("/MyAccount", "My account")]
         [InlineData("/MyAccount/EditDetails", "Edit details")]
+        [InlineData("/Signposting/LaunchLearningResource/3", "View resource \"Test image resource\"")]

```

Comment:
Looks like the automated UI tests are failing to pick up the API key when run locally, I'll have a look into it shortly.